packages/sanity/src/structure/panes/document/documentPanel/banners/__telemetry__/DraftLiveEditBanner.telemetry.ts

```diff
@@ -0,0 +1,21 @@
+import {defineEvent} from '@sanity/telemetry'
+
+/**
+ * When a draft in a live edit document is published
+ * @internal
+ */
+export const PublishedLiveEditDraft = defineEvent({
+  name: 'Publish a draft (live edit)',
+  version: 1,
+  description: 'User published a draft when a draft of a live edit document to continue editing',
+})
+
+/*
+ * When a draft in a live edit document is discarded
+ * @internal
+ */
+export const DiscardedLiveEditDraft = defineEvent({
+  name: 'Create a new draft',
```

Comment:
Same question as above. This event name in isolation indicates its just about creating any regular draft, and should maybe say that it's about resolving the issue